### PR TITLE
[#1] [Chore] Setup project - Part 1

### LINKED ISSUE #1: Set up project repository
## Why

When starting a new project, the project repository foundations need to be in placed for smooth application's development with proper app building and running tests guidance.

## Acceptance Criteria
- [ ] Add pull request and issue templates.
- [ ] Setup proper `.gitignore` file for the project.
- [ ] Setup all necessary 3rd party libraries such as: RxSwift, Alamofire, etc. and library management tools such as: CocoaPods, Swift Package Manager, etc.
- [ ] Run the script from iOS template to generate initial classes and folder structure or create manually.
- [ ] Setup `README.md` file with proper description, setup/testing instruction, features requirements and API specs links.

### PR DESCRIPTION
**Pull request by markgravity**
Resolved #1 

## What happened

When starting a new project, the project repository foundations need to be in place for smooth application development with proper app building and running tests guidance.

## Insight

- Remove all junk files from ios-template
- Add pull request and issue templates.
- Update gitignore
 
## Proof Of Work

n/a

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-https://github.com/nimbl3/ios-templates/issues/
+#
```
Comment (minhnimble) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: please use `https://github.com/nimblehq/nimble-medium-ios/issues/` instead of `#`

**Review by minhnimble — COMMENTED** [review]
The rest LGTM 👍

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (markgravity) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: I'm trying to use this style from our [rxswift-demo](https://github.com/nimblehq/mvvm-rxswift-demo/pull/74)
It much simple and cleaner. Maybe we can follow their pattern `Resolves #{number}`, what do you think?
Screen Shot 2021-08-06 at 10 28 14

**Review by minhnimble — APPROVED** [review]
LGTM 🚀

**Pull request merged**